apps/complaints/fields/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+var _ = require('lodash');
+
+module.exports = _.extend(
+
+);
```

Comment:
You can remove this. If you get an error, remove the line in router.js that require's this file